assets/css/application.scss

```diff
@@ -923,3 +923,109 @@ input.form-control.show-loader {
     //     left: 98%;
     // }
 }
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        scrollbar-color: #333 #000;
+    }
+
+    body,
+    #wrapper,
+    .panel-body,
+    .xt-toc {
+        color: #bbb;
+        background: #000;
+    }
+
+    .home-link:after,
+    [role=separator] {
+        display: none;
+    }
+
+    .form-label,
+    .dropdown-menu {
+        background: #222;
+    }
+
+    header {
+        background: #222 !important;
+        border: none !important;
+    }
+
+    h4,
+    a:hover,
+    .typeahead * {
+        color: #bbb !important;
+    }
+    .dropdown-menu a:hover {
+        background: #226 !important;
+    }
+
+    .navbar-default,
+    nav,
+    table,
+    .panel-primary,
+    .panel-default,
+    th,
+    td,
+    .stat-list--group > tr:first-child > td,
+    hr,
+    .form-label,
+    .form-control {
+        border: none !important;
```

Comment:
This makes adjacent input fields seem like they are one big textarea, and to me it's a little hard to read some tables without the borders. What do you think?

Reply:
Fixed, please check.